R/cloneVariable.R

```diff
@@ -30,6 +30,10 @@ cloneVariable.GADSdat <- function(GADSdat, varName, new_varName, label_suffix =
   if(new_varName %in% namesGADS(GADSdat)) {
     stop("'",  new_varName, "' is already an existing variable in the 'GADSdat'.")
   }
+  ## alternativley, cloneVariable() could be extended to accept new_varName == varName. The only action
```

Comment:
Interesting point.

I think my personal preference is to only have open issues that are actual _to dos_ (this is more of a vague note that might never be relevant) but I see that others may have different preferences.

For reference, I checked how base R is using comments, for instance [grep()](https://github.com/SurajGupta/r-source/blob/master/src/library/base/R/grep.R) (comments from line 166). For now, I will leave as it is with added `<Note>`. Maybe this could also be something for our code convention meeting?